client/my-sites/customer-home/cards/features/domain-upsell/index.jsx

```diff
@@ -41,9 +41,7 @@ export function RenderDomainUpsell() {
 	const siteSubDomain = siteSlug.split( '.' )[ 0 ];
 	const locale = useLocale();
 	const { allDomainSuggestions } =
-		useDomainSuggestions( siteSubDomain, 3, undefined, locale, {
-			vendor: 'domain-upsell',
-		} ) || {};
```

Comment:
Is removing this intentional?

Reply:
Definitely no. It mixed both PR I'm working on. Updating in a couple of minutes 🤦‍♂️